schemas/constructs/v1alpha1/capability.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$id": "https://schemas.meshery.io/capability.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Meshery manages entities in accordance with their specific capabilities. This field explicitly identifies those capabilities largely by what actions a given component supports; e.g. metric-scrape, sub-interface, and so on. This field is extensible. Entities may define a broad array of capabilities, which are in-turn dynamically interpretted by Meshery for full lifecycle management.",
+  "additionalProperties": false,
+  "type": "object",
+
+  "required": ["schemaVersion", "version", "displayName", "kind", "type", "state", "status"],
+
+  "properties": {
+    "schemaVersion": {
+      "$ref": "../core.json#/definitions/versionString",
+      "description": "Specifies the version of the schema to which the capability definition conforms."
+    },
+    "version": {
+      "$ref": "../core.json#/definitions/semverString",
+      "description": "Version of the capability definition.",
+      "minLength": 1,
+      "maxLength": 50
+    },
+    "displayName": {
+      "$ref": "../core.json#/definitions/inputString",
+      "description": "Name of the capability in human-readible format."
+    },
+    "description": {
+      "type": "string",
+      "description": "A written representation of the purpose and characteristics of the capability."
+    },
+
+    "kind": {
+      "$ref": "../core.json#/definitions/inputString",
+      "description": "Kind of the capability",
+      "anyOf": [
+        {
+          "const": "action",
+          "description": "For capabilities related to executing actions on entities"
+        },
+        {
+          "const": "mutate",
+          "description": "For capabilities related to mutating an entity"
+        },
+
+        {
+          "const": "interaction",
+          "description": "Catch all for capabilities related to interaction with entities"
+        }
+      ]
+    },
+
+    "type": {
+      "$ref": "../core.json#/definitions/inputString",
+      "description": "Classification of capabilities. Used to group capabilities similar in nature."
+    },
+
+    "subType": {
+      "$ref": "../core.json#/definitions/inputString",
+      "description": "Most granular unit of capability classification. The combination of Kind, Type and SubType together uniquely identify a Capability."
+    },
+
+    "key": {
+      "$ref": "../core.json#/definitions/inputString",
+      "description": "Key that backs the capability."
+    },
+
+    "state": {
+      "$ref": "../core.json#/definitions/inputString",
+      "description": "State of the entity in which the capability is applicable.",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": ["declaration", "instance"]
+      }
+    },
+
+    "status": {
+      "type": "string",
+      "description": "Status of the capability",
+      "enum": ["enabled", "disabled"]
```

Comment:
Good use of `status`. This property should become more or less a universal property in all Meshery constructs.

Reply:
Add a default value as "enabled"